samples/model_repository/vllm_model/config.pbtxt

```diff
@@ -26,6 +26,7 @@
 
 # Note: You do not need to change any fields in this configuration.
 
+name: "vllm_model"
```

Comment:
wasn't this just removed?

Reply:
https://github.com/triton-inference-server/vllm_backend/pull/8#discussion_r1355626598